GitUI/UserControls/RevisionGrid/RevisionGridControl.cs

```diff
@@ -887,6 +887,9 @@ public void PerformRefreshRevisions(Func<RefsFilter, IReadOnlyList<IGitRef>> get
             Lazy<IReadOnlyList<IGitRef>> getUnfilteredRefs = new(() => (getRefs ?? capturedModule.GetRefs)(RefsFilter.NoFilter));
             _ambiguousRefs = new(() => GitRef.GetAmbiguousRefNames(getUnfilteredRefs.Value));
 
+            // Get the "main" stash commit, including the reflog selector
+            Lazy<IReadOnlyCollection<GitRevision>> getStashRevs = new(() => (!AppSettings.ShowStashes ? new List<GitRevision>() : new RevisionReader(capturedModule, hasReflogSelector: true).GetStashes(cancellationToken)));
```

Comment:
```suggestion
            Lazy<IReadOnlyCollection<GitRevision>> getStashRevs =
                new(() => (!AppSettings.ShowStashes 
                    ? Array.Empty<GitRevision>()
                    : new RevisionReader(capturedModule, hasReflogSelector: true).GetStashes(cancellationToken)));
```

I also suspect that one set of parens is superficial.